### ISSUE TITLE
TemplateDoesNotExist at /setup_admin 

### ISSUE THREAD

**Issue by 14MR** [PostHog/posthog#1178]
**Describe the bug**
After executing `docker-compose -f docker-compose.dev.yml up` error occures: `TemplateDoesNotExist at /setup_admin  error`

**To Reproduce**
1. build docker cache, containers, volumes
2. `docker-compose -f docker-compose.dev.yml up`
3. go to `http://localhost:8000/`

**Expected behavior**
Some page is rendered

**Screenshots**
![image](https://user-images.githubusercontent.com/5824170/87044202-7fe43800-c1f6-11ea-8ddf-adde28338eaa.png)

**Hosted or self hosted?:**
- self-hosted
- master branch (commit 582ca4eed4d7206b7524b21169e9d1fc6b400da9)

**Additional context**

**Comment by timgl** [comment]
Hi! Can you try running `docker-compose -f docker-compose.dev.yml exec web python manage.py collectstatic`, then running `up` again?

**Comment by 14MR** [comment]
`FileNotFoundError: [Errno 2] No such file or directory: '/code/frontend/dist'`

Looks like frontend haven't built properly. I will try to rebuild everything and let you know.

**Comment by 14MR** [comment]
In frontend folder after rebuilding:
```
root@165fb7349eb3:/code/frontend# cd public/
root@165fb7349eb3:/code/frontend/public# ls
favicon-192x192.png  favicon-32x32.png  posthog-logo-white.svg  posthog-logo.png  posthog-logo.svg
```

In backend: 
```
root@165fb7349eb3:/code/posthog/templates# ls
auth_error.html  authorize_and_redirect.html  demo.html  login.html  registration  setup_admin.html  signup_to_team.html
```

staticfiles: 
```
root@165fb7349eb3:/code/staticfiles# ls
empty.txt
```

`yarn build` from Dockerfile successfully finished

**Comment by 14MR** [comment]
Get it working!
![image](https://user-images.githubusercontent.com/5824170/87047108-4f9e9880-c1fa-11ea-8d47-1389a7bd2d4f.png)

I changed `NODE_ENV` to  `development` in package.json:
```
        "build": "NODE_ENV=development webpack --config webpack.config.js...
```
And copied the static through `python manage.py collectstatic`

**Comment by ungless** [comment]
Fixed with #1200
